### ISSUE TITLE
Compile part of document, editing and reload

### ISSUE THREAD

**Issue by uzsolt** [vim-latex/vim-latex#85]
I think it would be nice when the following would work:
1. Select a part of document (`main.tex`).
2. Vim-latex opens a file (`part.tex`) with the content of the part.
3. Compile and check the results.
4. Make some changes (in `part.tex`).
5. Rebuild, check, it's okay.
6. Close the `part.tex` and its content inserted into `main.tex`.

The step 6. is missed now.

**Comment by gerw** [comment]
Yes, this sounds reasonable. However, it might be quite involved to get the last step right (in particular if `main.tex` changed in the meanwhile).

I will not have the time to add this feature. Pull requests are welcome.

**Comment by uzsolt** [comment]
I don't have time too. And I'm not expert in Vim-programming :(
I hope there will be a man who has time and not beginner in Vim-programming :)
